lab-06-chris/README.md

```diff
@@ -0,0 +1,10 @@
+paragraph about me:I am 36, Married for 17 years. I have two teenage boys 13 and 16. I joined the Air Force straight out of high school, spent 6 years as a fuels journeyman.I enjoy being outdoors, I love knowing the inner workings of everything, finish carpentry, and basically making stuff with my hands.
```

Comment:
Nice job with the read me. Generally the read me is more about the application and less about the dev. You'll want to discuss what you application is.